kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -388,3 +390,20 @@ fn collect_codegen_items<'tcx>(gcx: &GotocCtx<'tcx>) -> Vec<MonoItem<'tcx>> {
         }
     }
 }
+
+fn symbol_table_to_gotoc(tcx: &TyCtxt, file: &Path) {
```

Comment:
Can you please add a debug message here with the exact command that will be executed?

Reply:
done